test/e2e/storage/testsuites/volumelimits.go

```diff
@@ -191,12 +191,17 @@ func (t *volumeLimitsTestSuite) DefineTests(driver TestDriver, pattern testpatte
 		e2epod.SetAffinity(&selection, nodeName)
 		pod.Spec.Affinity = selection.Affinity
 		l.unschedulablePod, err = l.cs.CoreV1().Pods(l.ns.Name).Create(pod)
+		framework.ExpectNoError(err)
```

Comment:
could this error be annotated with what type of pod we could not create?

Reply:
Thanks @neolit123 
My mistake, this Pod (with one volume to exceed the limit) should not be created successfully.
Therefore, we can simply ignore this error.

Reply:
Updated!